tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,417 @@
+        "data": [
+            {
+                "header": {
+                    "signer_public_key": "03d425d2d17b64e3ef8fee028089a567fbb05bd556f98c0b6fb62bc5750ea62b8f",
+                    "transaction_ids": [
+                        "ea15ec70968c6cf23d44a15431e4f2eea190db377eee0ebcaa605adabbc7d205"
+                        "768a8088ea7b1c45cab73835b76e1f187be52fc06ca37deb975c693bfe7a0874",
+                    ],
+                },
+                "header_signature": "169c3691a4a78b476a1891bf2a7afac27bb218d182889443bfbbc4090bb933b1"
+                                    "0a8a9ea3866b5f478020831d7efa41c834903f2f9adc324d965b207714f72ed9",
+                "transactions": [
+                    {
+                        "header": {
+                            "batcher_public_key": "03d425d2d17b64e3ef8fee028089a567fbb05bd556f98c0b6fb62bc5750ea62b8f",
+                            "dependencies": [],
+                            "family_name": "sawtooth_settings",
+                            "family_version": "1.0",
+                            "inputs": [
+                                "000000a87cb5eafdcca6a8cde0fb0dec1400c5ab274474a6aa82c1c0cbf0fbcaf64c0b",
+                                "000000a87cb5eafdcca6a8cde0fb0dec1400c5ab274474a6aa82c12840f169a04216b7",
+                                "000000a87cb5eafdcca6a8cde0fb0dec1400c5ab274474a6aa82c1918142591ba4e8a7",
+                                "000000a87cb5eafdcca6a8f82af32160bc5311783bdad381ea57b4e3b0c44298fc1c14",
+                            ],
+                            "nonce": "",
+                            "outputs": [
+                                "000000a87cb5eafdcca6a8cde0fb0dec1400c5ab274474a6aa82c1c0cbf0fbcaf64c0b",
+                                "000000a87cb5eafdcca6a8f82af32160bc5311783bdad381ea57b4e3b0c44298fc1c14",
+                            ],
+                        },
+                        "header_signature": "ea15ec70968c6cf23d44a15431e4f2eea190db377eee0ebcaa605adabbc7d205"
+                                            "768a8088ea7b1c45cab73835b76e1f187be52fc06ca37deb975c693bfe7a0874",
+                        "payload": "CAESRAoic2F3dG9vdGgudmFsaWRhdG9yLmJhdGNoX2luamVj"
+                                   "dG9ycxIKYmxvY2tfaW5mbxoSMHgyNzVkYWUwZDZkODg4NjE5",
+                    },
+                ],
+            },
+        ],
+        "head": "2eeebae97b8a813ca64a76717a44de9fdd787342ed122e9e9399570be35ba354"
+                "279639ad2a5848b136891bcdac343e2628938e63dbf9dc0f3d2a0e63cc6cb51f",
+    }
+
+    mock_get_batch_by_ids = mocker.patch('cli.batch.service.loop.run_until_complete')
+    mock_get_batch_by_ids.return_value = expected_result
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--ids',
+        batch_id,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_result == json.loads(result.output).get('result')
+
```

Comment:
Create tests for valid, but non-existing parameters.

Reply:
Done.